apigee_devportal_kickstart.install

```diff
@@ -402,3 +403,154 @@ function apigee_devportal_kickstart_update_8007() {
   }
 
 }
+
+/**
+ * Fix image and categories fields for api_doc.
+ */
+function apigee_devportal_kickstart_update_8008() {
+  /** @var \Drupal\Core\Entity\EntityFieldManagerInterface $entity_field_manager */
+  $entity_field_manager = \Drupal::service('entity_field.manager');
+  /** @var \Drupal\Core\Field\FieldStorageDefinitionInterface $definitions */
+  $definitions = $entity_field_manager->getFieldDefinitions('node', 'apidoc');
+
+  /** @var \Drupal\Core\Entity\EntityDisplayRepository $entity_display_repository */
+  $entity_display_repository = \Drupal::service('entity_display.repository');
+  /** @var \Drupal\Core\Entity\Entity\EntityFormDisplay $entity_form_display */
+  $entity_form_display = $entity_display_repository->getFormDisplay('node', 'apidoc');
+  $entity_view_display = $entity_display_repository->getViewDisplay('node', 'apidoc', 'card');
+
+  $fields = [
+    'field_image_apidoc' => [
```

Comment:
The rest of the fields on this content type follow a format like `field_apidoc_XXX`, do you want to use that here? Just a minor note, thanks for the PR!

Reply:
We can't really do this here because each field has distinct form/view display config. 👍